### ISSUE TITLE
Unable to compile on macOS `/odyssey/sources/rules.c:791:4: error: expected expression before ')' token`

### ISSUE THREAD

**Issue by ruimarinho** [yandex/odyssey#332]
```
#9 81.48 Scanning dependencies of target odyssey
#9 81.52 [  2%] Building C object sources/CMakeFiles/odyssey.dir/daemon.c.o
#9 81.69 [  3%] Building C object sources/CMakeFiles/odyssey.dir/pid.c.o
#9 81.86 [  4%] Building C object sources/CMakeFiles/odyssey.dir/logger.c.o
#9 82.15 [  5%] Building C object sources/CMakeFiles/odyssey.dir/rules.c.o
#9 82.28 /odyssey/sources/rules.c: In function 'od_rules_merge':
#9 82.28 /odyssey/sources/rules.c:578:17: warning: unused parameter 'added' [-Wunused-parameter]
#9 82.28       od_list_t *added, od_list_t *deleted)
#9 82.28       ~~~~~~~~~~~^~~~~
#9 82.28 /odyssey/sources/rules.c: In function 'od_rules_validate':
#9 82.28 /odyssey/sources/rules.c:791:4: error: expected expression before ')' token
#9 82.28     ) {
#9 82.28     ^
#9 82.29 make[2]: *** [sources/CMakeFiles/odyssey.dir/build.make:102: sources/CMakeFiles/odyssey.dir/rules.c.o] Error 1
#9 82.29 make[1]: *** [CMakeFiles/Makefile2:198: sources/CMakeFiles/odyssey.dir/all] Error 2
#9 82.29 make: *** [Makefile:130: all] Error 2
```

**Comment by x4m** [comment]
Uh, really, that's a bug. Will fix soon.

**Comment by ruimarinho** [comment]
Fixed by #334. Thanks @x4m.
